pages/landing/patcher/contact/_form.html

```diff
@@ -16,9 +16,47 @@
   <input type="hidden" name="hello_gruntwork" value=""/>
   <input type="hidden" name="_landing-page" value="{{ site.url }}{{ page.url }}" />
   <input type="text" name="_gotcha" style="display:none" />
-  {% include_relative _input-text.html id="contact-name" label="Name" required=true placeholder="Jon Doe" %}
-  {% include_relative _input-text.html id="contact-email" label="Email" required=true placeholder="jon@acme.com" type="email" %}
+  {% include_relative _input-text.html id="contact-name" label="Name" required=true placeholder="Jane Doe" %}
+  {% include_relative _input-text.html id="contact-email" label="Email" required=true placeholder="jane@acme.com" type="email" %}
   {% include_relative _input-text.html id="contact-company" label="Company" required=true placeholder="Acme, Inc." %}
+  <div class="form-group">
+    <div class="row">
+      <div class="col-xs-12 col-md-4">
+        <label>Which programming languages are you interested in?</label>
```

Comment:
On the one hand, that's useful data. On the other, it's extra stuff for users to fill out, which might reduce form submissions. Let's try as-is, but we should sanity check that it isn't overwhelming for users.

Reply:
Also, minor tweak:

```suggestion
        <label>Which languages and tools do you use?</label>
```

Reply:
It looks like @marinalimeira removed this section and replace it with "Which plan are you interested in?" so I'll resolve this issue.